packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -39,6 +39,7 @@ const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
     }, []);
 
     const liveChatSetup = (is_logged_in: boolean) => {
+        window.LiveChatWidget.init();
```

Comment:
question: how will using asyncInit and initializing here solve the session leak problem?

Reply:
without including the asyncInit, application will call livechat once they landed in the environment after Oauth verification, then it will get the first page_url which contains sensitive token that can access into user account. By adding asyncInit, it will not render the function until LiveChatSetup in use-livechat.ts is called. In this way, the application will trigger the livechat call after user is successfully redirect into the application after Oauth.